### DISCUSSION TITLE
Welcome to VSLauncherX Discussions!

### DISCUSSION THREAD

**Discussion by Hefaistos68** [Hefaistos68/VSLauncherX#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sagianino** [comment]
Hi @Hefaistos68,
at first thanks to have shared your project it is very usefull.
I have implemented the support to the DarkMode using https://github.com/BlueMystical/Dark-Mode-Forms
I can create a pull request if you like it but, there is a different type of license and I'm an expert on that topic.
Before creating the pull request I would like to be sure that both licenses are compatible.

Here two screenshot of the result.

![Dark-MainScreen-sample](https://github.com/user-attachments/assets/79bb6352-3e34-4d74-8b97-696c5f5a3df5)

![Dark-Settings-sample](https://github.com/user-attachments/assets/87820bb6-1ff6-49d8-8a21-e7e81205d13d)

**Reply by Hefaistos68** [reply]
Looks pretty neat. What about an option to switch modes? Or only automatic from Windows settings?

**Reply by Hefaistos68** [reply]
Actually I would like to transition the whole app to WPF but I have neither time nor wanting as I am no expert in WPF.

**Reply by sagianino** [reply]
> Looks pretty neat. What about an option to switch modes? Or only automatic from Windows settings?

Sorry for delay in reply, but I had a very busy week.

Currently there isn't an option and it follows automatically Windows settings.
I think that it is possible to add an option, in case.

**Reply by sagianino** [reply]
> Actually I would like to transition the whole app to WPF but I have neither time nor wanting as I am no expert in WPF.

Moving to WPF can be a good idea, even if WPF seems to be not more in Microsoft plan.
There are a lot of framework that can be helpfull to achieve the goal.

Currently I mantain and develop an desktop application in WPF so, I know something about WP, but I don't have a lot of free time currently.
